src/app/modules/core/core.component.ts

```diff
@@ -9,14 +9,27 @@ import { Router } from '@angular/router';
 })
 export class CoreComponent implements OnInit {
     title = 'Inbox';
-    constructor(private _router: Router) { }
+    constructor(private _router: Router, private access: LoginService) { }
 
     ngOnInit(): void {
         this.title = 'Inbox';
+        this.access.verifyAccess().subscribe((data) => {
+            if (data.status === false) {
```

Comment:
sir i have replaced it with (!data.status) . because server is either returning true or false in status field